common/source/docs/common-polygon_fence.rst

```diff
@@ -0,0 +1,65 @@
+.. _polygon_fence:
+
+[copywiki destination="copter,rover"]
+================
+Polygon GeoFence
+================
+
+Overview
+========
+
+Copter 3.4 (and higher) include support for a polygon geofence with up to 70 points.  The purpose of this fence is to attempt to stop your vehicle from flying out of the polygon by initiating an RTL or, if flying in Loiter mode, the vehicle will normally stop before breaching the fence.  This feature is an extension of the simpler :ref:`cylindrical fence <common-ac2_simple_geofence>`.
```

Comment:
Maybe remove this "Copter-3.4" part because there is a note below providing the version.

Reply:
done